media/windows/ricoh-printer/ricoh_db_create.py

```diff
@@ -0,0 +1,121 @@
+#!/user/bin/env python3
+
+# Script which runs daily to update staff print totals
+# based on CSVs generated by the printer itself.
+
+import sys
+sys.path.insert(1, '../../python/')
+
+import os
+import re
+import csv
+import datetime
+import sqlite3
+from sqlite3 import Error
+
+#import ECC
+#import PDSChurch
+
+from pprint import pprint
+from pprint import pformat
+
+###########################################################
+
+ricoh_db = "media/windows/ricoh-printer/ricoh.db"
+today = datetime.date.today()
+datestring = f'{today.year}{today.month}{today.day}'
+debug_datestring = '20201013'
+
+def _strip(value):
+    #The CSV puts brackets around values, which we don't want
+    return value.strip('[').strip(']')    
+
+def load_latest_csv():
+    
+    filename = f'RICOH IM C4500_usercounter_{datestring}.csv'
+    
+    debug_filename = 'media/windows/ricoh-printer/RICOH IM C4500_usercounter_20201013.csv'
+    
+    csv_rows = list()
+
+    with open(debug_filename, encoding='utf-8') as csvfile: #TODO: change to filename when retrieval is automated
```

Comment:
For debugging purposes, it's best to try to have as minimal an impact as possible.  I.e., don't use new variable names that have to be edited/removed later.  Instead, write the code in as "final" a form as possible, but have a noticeable / obvious single place where you set debug values.  For example:

```python
def load_csv(filename):
    with open(filename, encoding='utf-8') as csvfile:
        csvreader = csv.DictReader(csvfile)
        # ...etc.

def main():
    args = setup_cli_args()

    csv_filename = args.csv

    # HC DEBUG: Temporarily override the filename with a hard-coded
    # filename for debugging/development purposes.
    csv_filename = 'f'RICOH IM C4500_usercounter_{datestring}.csv''

    data = load_csv(csv_filename)
    # ...etc.
```